src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -123,6 +130,20 @@ export const ContactPolicy = InjectAppServices(
 
       return errors;
     };
+    
+    const getSubscriberToAdd = () => {
+      const subscriberId = Math.floor(Math.random() * 21);
+      return {
+        id: subscriberId,
+        [labelKey]: `subscriber${subscriberId}`,
+      };
+    };
```

Comment:
Here the this that we will be adding into the cloudTag is a List, and not a Subscriber.
```suggestion
    const getListToAdd = () => {
      const listId = Math.floor(Math.random() * 21);
      return {
        id: listId,
        [labelKey]: `List${listId}`,
      };
    };
```

Reply:
Sometimes we use List or SubscribersList to define a List.